src/supervisor/watchers/handlers/cron-job.ts

```diff
@@ -27,6 +32,24 @@ export async function paginatedCronJobList(namespace: string): Promise<{
   );
 }
 
+export async function paginatedCronJobV1Beta1List(namespace: string): Promise<{
+  response: IncomingMessage;
+  body: V1beta1CronJobList;
+}> {
+  const cronJobList = new V1beta1CronJobList();
+  cronJobList.apiVersion = 'batch/v1beta1';
+  cronJobList.kind = 'CronJobList';
+  cronJobList.items = new Array<V1beta1CronJob>();
```

Comment:
If we are specifying this then to be accurate we need to change on line 53 the input to accept one or the other type:

```typescript
export async function cronJobWatchHandler(cronJob: V1CronJob | V1beta1CronJob): Promise<void> {
```
Because both v1 and v1beta1 share the same DELETE handler